src/Microsoft.Framework.Runtime/Project.cs

```diff
@@ -188,6 +190,21 @@ internal static Project GetProjectFromStream(Stream stream, string projectName,
                 }
             }
 
+            var fileVersion = Environment.GetEnvironmentVariable("DNX_ASSEMBLY_FILE_VERSION");
+            if (string.IsNullOrWhiteSpace(fileVersion))
+            {
+                fileVersion = project.Version.ToString();
```

Comment:
Why to ToString and new Verison right underneath? If you're defaulting to the project.Version then just assign it directly.